app/validators/claim/transfer_claim_sub_model_validator.rb

```diff
@@ -19,14 +15,10 @@ def has_many_association_names_for_steps
       case_details: [],
       defendants: %i[defendants],
       offence_details: [],
-      fees: %i[
-        misc_fees
-        disbursements
-        expenses
-        messages
```

Comment:
[Ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2252/files#r170615857)